etc/config.xml

```diff
@@ -45,6 +45,7 @@
             </bold_checkout_payment_booster>
             <bold_checkout_payment_booster_advanced>
                 <api_url>https://api.boldcommerce.com/</api_url>
+                <eps_url>https://eps.secure.staging.boldcommerce.com/</eps_url>
```

Comment:
Using staging url right now as prod is not available yet. This existence of this option is temporary and will eventually be removed once the SPI SDK takes over the communication with EPS.